examples/closed-issue-counter/modules/github/client.bal

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+final string GIT_GRAPHQL_API_URL = "https://api.github.com/graphql";
+final string EMPTY_STRING = "";
+
+public client class Client {
+
+    string accessToken;
+    http:Client githubGraphQlClient;
+
+    public isolated function init(Configuration config) returns error? {
+        self.accessToken = config.accessToken;
+        self.githubGraphQlClient = check new(GIT_GRAPHQL_API_URL, config.clientConfig);
+    }
+
+    remote isolated function getRepositoryIssueList(string repositoryOwnerName, string repositoryName,
```

Comment:
```suggestion
    remote isolated function getClosedIssueList(string repositoryOwnerName, string repositoryName,
```

Reply:
Shall we rename the function to `getClosedIssueList`, as it always return closed issue list